README.md

```diff
@@ -6,6 +6,23 @@ REST APIs using Jersey, Jackson, Jetty, and Hibernate Validator.
 
 See the [`examples/`](examples/) directory for a simple demo service.
 
+Security
+--------
+The REST `Application` can support both http and https. https is disabled by default. Three modes are supported:
+http, https, and http+https. In the last mode, http and https are supported on different ports.
```

Comment:
The way rest-utils is currently implemented, it does not specify which host to listen on, so all network interfaces are listened on. But Jetty does support listening on specific interfaces via `NetworkTrafficServerConnector.setHost()`. I like the `listeners` model. Going this route will also let the end-user listen on specific interfaces. I'll make this change -- good suggestion, @ewencp!